cfg/builder/builder_walk.cc

```diff
@@ -426,6 +426,7 @@ BasicBlock *CFGBuilder::walk(CFGContext cctx, ast::ExpressionPtr &what, BasicBlo
                         target.isRepeated = e.flags.isRepeated;
                         target.isDefault = e.flags.isDefault;
                         target.isShadow = e.flags.isShadow;
+                        target.isBlock = e.flags.isBlock;
```

Comment:
Should we just make this whole thing `argFlags.emplace_back(e.flags)`?

Reply:
I like that, thanks!